Automattic/wp-calypso
# Fix bad Odie chat input position (when system scrollbar is visible)

Fixes https://github.com/Automattic/dotcom-forge/issues/8614

## Proposed Changes

* Position `.odie-chat-message-input-container` so it's not pushed outside its container when system scrollbars are visible

| Before | After |
| ------ | ----- |
| ![before](https://github.com/user-attachments/assets/1f0ca0cf-1586-4c4b-907b-3ac1c8889027) | ![after](https://github.com/user-attachments/assets/886a9d9d-ef3b-498c-8497-16fa6daee55e) |

## Testing Instructions



1. On your Mac, open System Settings > Appearance and change "Show scroll bars" to "Always"
2. Navigate to `/help`
3. Click `Get help`
4. Say something that makes Wapuu generate a long answer (for example `my ssl certificate doesn't work`)
5. Ensure that the input wrapper is positioned correctly
